LEAF_Request_Portal/utils/db_upgrade/Update_RMC_DB_2020041300-2020071600.sql

```diff
@@ -0,0 +1,9 @@
+START TRANSACTION;
+
+ALTER TABLE `users` 
+ADD COLUMN `locallyManaged` TINYINT(1) NULL DEFAULT 0 AFTER `primary_admin`,
+ADD COLUMN `active` TINYINT(4) NOT NULL DEFAULT 1 AFTER `locallyManaged`;

```

Comment:
This file needs to be renamed to; this is the latest db_upgrade file Update_RMC_DB_202062600-2020071700.sql FYI